### ISSUE TITLE
openni wrapper does not publish depth/image

### ISSUE THREAD

**Issue by cdondrup** [strands-project/scitos_robot#34]
Is there a reason for the new openni wrapper not to publish /camera_ns/depth/image but just image_raw and image_rect? Also image_raw does not seem to use image_transport and is therefore not compressed.
Not having much to do with vision in general I don't really know what the difference between image and image_raw is so maybe I can just use the raw one?

**Comment by RaresAmbrus** [comment]
image_raw is the unprocessed output of the camera, and image_rect is the rectified raw image, taking into account the camera intrinsic parameters. I'm not sure what the /camera_ns/depth/image in the old openni driver is, but I think the raw and rectified image is all we need. 
You're right, the driver node doesn't use image_transport, I should add that. But it's implemented as a nodelet, and that helps when transferring the image, since everything else is a nodelet in the rest of the processing pipeline (rectifying, registering, converting to a point cloud).

**Comment by cdondrup** [comment]
It's very comforting that you also don't know what the image topic does. Makes me feel less stupid.

I agree that it does not make a difference for the internal nodelet processing if the raw image is not published using image_transport but as soon as someone is to stream it to another PC (for whatever reason) it would be very nice to have. So if it is not too much of an effort, I think it would be good.

**Comment by RaresAmbrus** [comment]
Yep, agreed. I'm a bit busy now but I'll try to do it by the end of the week. In the mean time one can stream one of the other topics, like the rectified image, to get image compression.

**Comment by cdondrup** [comment]
Thank you.
Once that is finished you're welcome to close this issue.

**Comment by cburbridge** [comment]
@cdondrup Can this be closed?

**Comment by cdondrup** [comment]
The initial issue is solved but the raw images are still not published in compressed form. But I think no one is using them anyway. I will close this. Can be reopened in the new repository if someone needs this.
